ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -194,6 +209,21 @@ open class PopupMenuController: DrawerController {
         view.isHidden = true
         return view
     }()
+    private lazy var searchView: UIView = {
+        let view = UIStackView()
+        view.axis = .vertical
+        view.addArrangedSubview(searchBar)
+        view.isHidden = searchBar.isHidden
+        view.isLayoutMarginsRelativeArrangement = true
+        view.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 12, right: 16)
+        return view
+    }()
+    private let searchBar: SearchBar = {
```

Comment:
does your pod spec need to be updated for new dependencies?

Reply:
Yep added that in the new update but I'm wondering if we need to add the entirety of the Nav subspec or if we should create a new one just for Search, thoughts?

Reply:
Oh interesting! Hm would it be any better to add the dependency as a file path (ie. SearchBar folder path) inside the pod spec for Popup, or add the Navigation subspace entirely? Not sure if I want to factor Searcher into its own subspace in this PR.

Reply:
Probably also not a great solution, but it might make sense to make this just a different control so that we don't force a dependency on anything navigation or search bar on someone that just wants a plain popup menu.

Reply:
Different control as in subclass of PopupMenuController with a search bar instead of adding the search bar to the PopupMenuController.

Reply:
Hmmm... I don't know how I feel to be honest especially because we have another addition to the Popup coming up (pill button bar). Would that mean a subclass for every popupmenu variant? Or maybe a "search/filter" version of popup that contains these add-ons? And then there's also binary size to consider I suppose?

Reply:
I think splitting out to a search/filter version makes sense. I imagine the search bar and pill button bar for filtering will be used together, but there's lots of times where someone would want just a popup menu without any of the extra bits and dependencies adding those to the base class would add.